jerry-core/jerry-snapshot.h

```diff
@@ -24,27 +24,17 @@
  */
 typedef struct
 {
-  uint32_t last_compiled_code_offset; /**< offset of the last compiled code */
+  uint32_t version; /**< version number */
+  uint32_t lit_table_offset; /**< offset of the literal table */
   uint32_t lit_table_size; /**< size of literal table */
-  __extension__ uint32_t is_run_global : 1; /**< flag, indicating whether the snapshot
-                                             *   was dumped as 'Global scope'-mode code (true)
-                                             *   or as eval-mode code (false) */
+  uint8_t is_run_global; /**< flag, indicating whether the snapshot
```

Comment:
Why not simply use the type `bool` here? I don't see any alignment requirements or need for multiple flags that would necessitate an integer type with fixed bit-length.